eventdiscovery-sdk/src/main/res/values/dimens.xml

```diff
@@ -14,5 +14,7 @@
     <dimen name="schedjoules_event_list_progress_bar">36dp</dimen>
     <dimen name="schedjoules_event_detail_item_icon">28dp</dimen>
 
+    <dimen name="schedjoules_location_list_item_height">62dp</dimen>
```

Comment:
Updated to 64dp, also corrected the layout to align the divider correctly to the bottom.